volatility3/framework/plugins/windows/amcache.py

```diff
@@ -14,8 +14,11 @@ class Amcache(amcache.Amcache):
     _required_framework_version = (2, 0, 0)
     _version = (2, 0, 0)
 
-    def __getattr__(self, *args, **kwargs):
+    def __getattribute__(self, *args, **kwargs):
         warnings.warn(
-            FutureWarning("This plugin is now called windows.registry.amcache.Amcache")
+            FutureWarning(
+                "The windows.amcache.Amcache plugin is deprecated and will be removed on "
+                "September 19, 2025. Use windows.registry.amcache.Amcache instead."
```

Comment:
Sorry, don't mean to be picky, but can we use 2025-09-15, just to keep it consistent with the other deprecation messages (and make it a bit easier to grep for)?

Reply:
No problem, that's done. One other question.  Is there a reason the certificates.py is in volatility3/plugins/windows/registry and not volatility3/framework/plugins/windows/registry? Should we move it into the framework directory as part of this?

Reply:
Yeah, it was supposed to be an example of a contributed/not supported plugin.  To show people where they're supposed to put plugins they write.  There's another plugin lurking under there too.  I dunno if anyone ever really realized, but I'd like to keep them there, since the intention was people would put their plugins in the main plugin directory, not the core plugins (they can overload core plugins that way too, since it takes precedence).